.github/workflows/zizmor.yaml

```
@@ -0,0 +1,37 @@
+# https://github.com/woodruffw/zizmor
+name: GitHub Actions Security Analysis with Zizmor
+
+on:
+  push:
+    branches: ["master", "main"]
+  pull_request:
+    branches: ["**"]
```

Small thing: shouldn't this be `*`?

```suggestion
    branches: ["*"]
```

---

Ah nvm, TIL `**` means "match including `/`".

---

I'm not certain. I think that made its way into my example repo's config when I copy pasted somebody else's. Let's ship it as a fix and fix it later if it doesn't work I guess.